inc/integrations/class-pico.php

```diff
@@ -64,6 +64,17 @@ public function setup() {
 		}
 
 		add_filter( 'wp_irving_verify_coral_user', [ $this, 'verify_pico_user_for_sso' ] );
+
+		// Use staging API urls.
+		if ( get_option( 'irving_integrations' )['pico']['use_staging'] ?? false ) {
+			if ( ! defined( 'PP_API_ENDPOINT' ) ) {
+				define( 'PP_API_ENDPOINT', 'https://api.staging.pico.tools' ); // phpcs:ignore
```

Comment:
What phpcs rule is being ignored here?  I don't see an issue...

Reply:
It wants `PP_API_ENDPOINT` to have a WP Irving namespace.